modules/javafx.graphics/src/main/java/javafx/scene/layout/GridPane.java

```diff
@@ -2607,21 +2627,17 @@ private double getSize(int position) {
             return singleSizes[position];
         }
 
-        private void setMaxSize(int position, double size) {
-            singleSizes[position] = Math.max(singleSizes[position], size);
+        private boolean hasGapBefore(int position) {
+            return gapBefore[position];
         }
 
-        private void setMultiSize(int startPosition, int endPosition, double size) {
-            if (multiSizes == null) {
-                multiSizes = new TreeMap<>();
-            }
-            Interval i = new Interval(startPosition, endPosition);
-            multiSizes.put(i, size);
+        private void setMaxSize(int position, double size) {
+            singleSizes[position] = Math.max(singleSizes[position], size);
```

Comment:
I don't know why github shows this as a change.
Unused `setMultiSize()` has been removed which is ok.